code/game/turfs/simulated/floor/plating.dm

```diff
@@ -106,6 +107,16 @@
 			new /obj/item/stack/rods(src, 2)
 			ChangeTurf(/turf/simulated/floor/plating)
 			return
+	if(istype(C, /obj/item/stack/sheet/plasteel) && insulated == 0) //Insulating the floor
+		to_chat(user, "<span class='notice'>You begin insulating [src]...</span>")
+		if(do_after(user, 40, target = src) && insulated == 0) //You finish insulating the insulated insulated insulated insulated insulated insulated insulated insulated vacuum floor
+			to_chat(user, "<span class='notice'>You finish insulating [src].</span>")
+			var/obj/item/stack/sheet/plasteel/W = C
+			W.use(1)
+			thermal_conductivity = 0.0
```

Comment:
Just change this to 0, BYOND has no difference between ints and floats, they're all floats underneath